### PR TITLE
Analytics Rules minor changes

### PR DESCRIPTION
**Pull request by vakohl**
Required items, please complete
   
   Change(s):
* Description
* use of contains
* Updated entity mapping
* removed old Column mapping

   Reason for Change(s):
   - Rule re-validation

   Version Updated:
   - Various versions

   Testing Completed:
   - Syntactical testing done only. Data wasn't available for testing.

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/Azure Activity/Analytic Rules/AADHybridHealthADFSNewServer.yaml [hunk 1]
```diff
@@ -24,21 +24,21 @@ query: |
   AzureActivity
   | where CategoryValue == 'Administrative'
```
Comment (petebryan) [inline Solutions/Azure Activity/Analytic Rules/AADHybridHealthADFSNewServer.yaml:25]: We should update this one to =~ as well

File: Solutions/Azure Activity/Analytic Rules/NRT-AADHybridHealthADFSNewServer.yaml [hunk 2]
```diff
@@ -20,20 +20,21 @@ query: |
   AzureActivity
   | where CategoryValue == 'Administrative'
```
Comment (petebryan) [inline Solutions/Azure Activity/Analytic Rules/NRT-AADHybridHealthADFSNewServer.yaml:21]: =~

**Review by petebryan — CHANGES_REQUESTED** [review]

File: Solutions/Azure Activity/Analytic Rules/AADHybridHealthADFSNewServer.yaml (hunk 1 above)
Comment (vakohl) [reply Solutions/Azure Activity/Analytic Rules/AADHybridHealthADFSNewServer.yaml:25]: thanks @petebryan  this is done.

File: Solutions/Azure Activity/Analytic Rules/AADHybridHealthADFSServiceDelete.yaml [hunk 3]
```diff
@@ -25,21 +25,21 @@ query: |
   AzureActivity
   | where CategoryValue == 'Administrative'
```
[the remark this answers is not in the document]
Comment (vakohl) [reply Solutions/Azure Activity/Analytic Rules/AADHybridHealthADFSServiceDelete.yaml:26]: thanks. This is done.

File: Solutions/Azure Activity/Analytic Rules/AADHybridHealthADFSSuspApp.yaml [hunk 4]
```diff
@@ -29,22 +29,22 @@ query: |
   AzureActivity
   | where CategoryValue == 'Administrative'
```
[the remark this answers is not in the document]
Comment (vakohl) [reply Solutions/Azure Activity/Analytic Rules/AADHybridHealthADFSSuspApp.yaml:30]: thanks. this is done

File: Solutions/Azure Activity/Analytic Rules/NRT-AADHybridHealthADFSNewServer.yaml (hunk 2 above)
Comment (vakohl) [reply Solutions/Azure Activity/Analytic Rules/NRT-AADHybridHealthADFSNewServer.yaml:21]: done

**Review by petebryan — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**